EIPS/7639.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 7639
+title: Cease serving history before PoS
+description: Execution layer clients will no longer serve block data before Paris over p2p.
+author: lightclient (@lightclient)
+discussions-to: https://ethereum-magicians.org/t/cease-serving-history-before-pos/18991
+status: Draft
+type: Standards Track
+category: Networking
+created: 2024-02-13
+---
```

Comment:
Maybe add dependency on eip-6110?

Reply:
I don't think this is dependent on 6110? This only removes pre-merge data. 6110 only modifies the behavior of exits after Prague, so clients still need to sort out how to deal with the loss of pre-merge receipts.